latest/CCSP/EXAMPLE/example_custom_product.yaml

```
@@ -0,0 +1,47 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/CCSP/EXAMPLE/example_custom_product-1.0.0
+
+title: Example CCSP custom product
+
+datamodel_name: ExampleCustomProductModel
+
+type: object
+properties:
+  meta:
+    allOf:
+      - $ref: asdf://stsci.edu/datamodels/roman/schemas/CCSP/ccsp_custom_product-1.0.0
+      - required: [target_coordinates]
+      - type: object
+        properties:
+          wcs:
+            title: World Coordinate System (WCS)
+            tag: tag:stsci.edu:gwcs/wcs-*
+          widget:
+            title: Widget metadata
+            properties:
+              state:
+                title: Widget state
+                type: string
+              count:
+                title: Widget count
+                type: integer
+            required: [state, count]
+        required: [wcs, widget]
+  data:
+    title: My data
+    description: Described here
+    tag: tag:stsci.edu:asdf/core/ndarray-1.*
+    datatype: float32
+    exact_datatype: true
+    unit: "DN"
+  err:
+    title: Optional err
+    description: Described here
+    tag: tag:stsci.edu:asdf/core/ndarray-1.*
+    datatype: float32
+    exact_datatype: true
+    unit: "DN"
```

```suggestion
  data:
    title: Science Data (MJy/sr)
    description: Described here
    tag: tag:stsci.edu:asdf/core/ndarray-1.*
    datatype: float32
    exact_datatype: true
    ndim: 2
    unit: "MJy/sr"
  err:
    title: Error (MJy/sr)
    description: Total error array corresponding to data
    tag: tag:stsci.edu:asdf/core/ndarray-1.*
    datatype: float32
    exact_datatype: true
    ndim: 2
    unit: "MJy/sr"
```
(Just for encouraging good habits in the PITs in descriptive specificity, and switching to a more familiar unit.)